unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/BuilderProjectsPanel/Scripts/Views/ProjectCardView.cs

```diff
@@ -1,52 +1,322 @@
+
+        rectTransform = GetComponent<RectTransform>();
+    }
+
+    private void OnDestroy()
+    {
+        isDestroyed = true;
+        Dispose();
+    }
+
+    public void Dispose()
+    {
+        AssetPromiseKeeper_Texture.i.Forget(thumbnailPromise);
+        editorButton.onClick.RemoveAllListeners();
+        if(!isDestroyed)
+            Destroy(gameObject);
+    }
+    
+    public void Setup(ProjectData projectData)
+    {
+        this.projectData = projectData;
+        SetName(projectData.title);
+        SetSize(projectData.rows,projectData.colums);
+        
+        SetThumbnail(projectData.id,projectData.thumbnail);
+        
+        ((IProjectCardView)this).searchInfo.SetId(projectData.id);
+    }
+
+    public void SetScenes(List<Scene> scenes)
+    {
+        scenesDeployedFromProject = scenes;
+        publishedGameObject.gameObject.SetActive(true);
+        if (scenes.Count == 0)
+        {
+            syncImage.enabled = false;
+            projectSyncTxt.text = "NOT PUBLISHED";
+            downButtonTransform.gameObject.SetActive(false);
+        }
+        else
+        {
+            downButtonTransform.gameObject.SetActive(true);
+            bool isSync = true;
+            long projectTimestamp = BIWUtils.ConvertToMilisecondsTimestamp(projectData.updated_at);
+            foreach (Scene scene in scenes)
+            {
+                if (scene.deployTimestamp < projectTimestamp)
+                {
+                    isSync = false;
+                    break;
+                }
+            }
+            if (isSync)
+                syncImage.color = syncColor;
+            else
+                syncImage.color = desyncColor;
+            
+            projectSyncTxt.text = "PUBLISHED IN";
+        }
+    }
+
+    private void ExpandButtonPressed()
+    {
+        if(scenesDeployedFromProject.Count == 0 )
+            return;
+        
+        downButtonTransform.Rotate(Vector3.forward,180);
+        
+        scenesAreVisible = !scenesAreVisible;
+        ScenesVisiblitityChange(scenesAreVisible);
+        
+        float amountToIncrease = sceneCardViews.Count * SCENE_CARD_SIZE + SCENE_CARD_TOTAL_PADDING + SCENE_CARD_ITEM_PADDING * (sceneCardViews.Count-1);
+        
+        if (scenesAreVisible)
+        {
+            layoutGroup.padding.bottom = 18;
+            AddRectTransformHeight(rectTransform,amountToIncrease);
+            AddRectTransformHeight(scenesContainer,amountToIncrease);
+        }
+        else
+        {
+            layoutGroup.padding.bottom = 0;
+            AddRectTransformHeight(rectTransform, -amountToIncrease);
+            AddRectTransformHeight(scenesContainer, -amountToIncrease);
+        }
+
+        OnExpandMenuPressed?.Invoke();
+    }
+
+    private void AddRectTransformHeight(RectTransform rectTransform, float height)
+    {
+        CoroutineStarter.Start(ChangeHeightAnimation(rectTransform, height));
```

Comment:
It is not needed the use of `CoroutineStarter.Start()` in MonoBehaviours code, in this case is better to use `Coroutine.Start()`.

Reply:
AFAIK @BrianAmadori told to me use the CouroutineStarter always since sometimes the coroutines could end in a memory leak. 

Brian should we maintain this direction?